package.json

```diff
@@ -16,9 +16,8 @@
     "backstop:clean": "rm -rf backstop_data/bitmaps_test backstop_data/html_report",
     "backstop:docker:test": "backstop test --docker --config=backstop_data/backstop_test.js --env=local",
     "backstop:ci:test": "backstop test --docker --config=backstop_data/backstop_test.js --env=ci",
+    "backstop:test-all": "backstop test --docker --config=backstop_data/backstop_test.js --env=local themes=lightest,light,dark,darkest scales=medium,large",
```

Comment:
this PR might need a rebase @lazd . The rest of the PR looks great.
- removed alphas and updated script to do semver automatically for packages, right?

Reply:
Looks like that was just renaming the `prepublishOnly` script. We're good!